itest/litd_custom_channels_test.go

```diff
@@ -482,14 +482,12 @@ func testCustomChannels(_ context.Context, net *NetworkHarness,
 	// a direct channel invoice payment with no RFQ SCID present in the
 	// invoice.
 	// ------------
-	paidAssetAmount := createAndPayNormalInvoice(
+	createAndPayNormalInvoice(
 		t.t, charlie, dave, dave, 20_000, assetID, withSmallShards(),
+		withFailure(lnrpc.Payment_FAILED, lnrpc.PaymentFailureReason_FAILURE_REASON_INCORRECT_PAYMENT_DETAILS),
```

Comment:
lll, here and below. Can use `failureIncorrectDetails` that's defined in `assets_test.go` instead.